aws-rds-dbclusterendpoint/aws-rds-dbclusterendpoint.json

```diff
@@ -0,0 +1,139 @@
+{
+  "typeName": "AWS::RDS::DBClusterEndpoint",
+  "description": "The AWS::RDS::DBClusterEndpoint resource allows you to create custom Aurora Cluster endpoint. For more information, see Using custom endpoints in the Amazon RDS Aurora Guide.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds",
+  "tagging": {
+    "taggable": true
+  },
+  "replacementStrategy": "delete_then_create",
+  "definitions": {
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The key name of the tag. You can specify a value that is 1 to 128 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -.",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "description": "The value for the tag. You can specify a value that is 0 to 256 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -.",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "DBClusterIdentifier": {
+      "description": "The DB cluster identifier of the DB cluster associated with the endpoint. This parameter is stored as a lowercase string.",
+      "type": "string",
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,62}$",
+      "minLength": 1,
+      "maxLength": 63
+    },
+    "DBClusterEndpointIdentifier": {
+      "description": "The identifier to use for the new endpoint. This parameter is stored as a lowercase string.",
+      "type": "string",
+      "pattern": "^[a-z]{1}(?:-?[a-z0-9]){0,62}$",
+      "minLength": 1,
+      "maxLength": 63
+    },
+    "EndpointType": {
```

Comment:
What about `CustomEndpointType`? Can we add it as `ReadOnly`parameter? Or it is redundant in all case and have same value of `EndpointType`?